__tests__/app.test.js

```diff
@@ -65,23 +65,70 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments[0]
+
+                expect(result).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
+                })
+
+            })
+        });
+
+        test("should return an array of objects in order", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
 
+                expect(result).toBeSortedBy('created_at', {
+                    descending: true,
+                    coerce: true,
+                });
+            })
         });
 
+        test("should return 404 when valid but non existent id is passed", () => {
+            return request(app).get("/api/articles/1000/comments").expect(404).then(({ body }) => {
+                expect(body.msg).toBe('Not Found')
+
+            })
+        })
+
+        test("should return object with expected length", () => {
+            return request(app).get("/api/articles/3/comments").expect(200).then((res) => {
+
+                const result = res.body.comments
+                expect(typeof result).toBe("object")
+                expect(result).toHaveLength(2)
```

Comment:
It's better to use [isArray](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) to check if something is an array, because both an object and an array would return true for this test (because technically arrays are objects)